src/dataset/DatasetTree.ts

```diff
@@ -685,31 +688,31 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
             favPrefix = node.label.substring(0, node.label.indexOf(":") + 2);
             beforeDataSetName = node.label.substring(node.label.indexOf(":") + 2);
         }
-        const afterDataSetName = await vscode.window.showInputBox({value: beforeDataSetName});
+        const afterDataSetName = await vscode.window.showInputBox({ value: beforeDataSetName });
```

Comment:
Similar to my comment for `renameDataSetMember` - are there any downsides to just doing`toUpperCase()` at the point when `afterDataSetName` is assigned (rather than throughout the function)?

Reply:
Hi Lauren, this is the same issue as the comment above. Please refer to that response 😄